spanner/client.go

```diff
@@ -177,6 +177,7 @@ func NewClientWithConfig(ctx context.Context, database string, config ClientConf
 			),
 		),
 		option.WithGRPCConnectionPool(config.NumChannels),
+		option.WithUserAgent(clientUserAgent),
```

Comment:
I think this will be overridden by any `user-agent` that a user has specified in the supplied options here: https://github.com/googleapis/google-cloud-go/blob/be88d0793e6c0825050037140e8c3d95e0cd0fed/spanner/client.go#L182-L184

Is that a problem?

Reply:
This should be good for us. Thanks for catching this!